### PR TITLE
dmctl: add exit code for command mode

### LINKED ISSUE #841: Exit code should not be 0 if failed to run command
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.
```
root@control:/tiup-cluster/tests/tiup-dm/script/task# dmctl --master-addr n1:8261 operate-source create ./source1.tomldfdf
get file content error:
open ./source1.tomldfdf: no such file or directory
root@control:/tiup-cluster/tests/tiup-dm/script/task# echo $?
0
```
2. What did you expect to see?

exit code should not be 0

3. What did you see instead?

exit code is 0 which means the command run successfully.

4. Versions of the cluster

    - DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):

```
Release Version: v2.0.0-beta.2-6-gc8e0913
Git Commit Hash: c8e09137b049207adf136a2931a370727c9ba109
Git Branch: master
UTC Build Time: 2020-06-28 12:38:45
Go Version: go version go1.13 linux/amd64
```

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
fix #841 

### What is changed and how it works?
change to use `RunE` of `cobra.Command`. besides, error will output to console by `github.com/spf13/cobra`, so refine some print log

### Check List 

Tests 

 - Integration test
    modify `dmctl_command`, add `... && exit 1 || echo "exit code should be not zero"`

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: dm/ctl/ctl.go [hunk 1]
```diff
@@ -131,11 +132,12 @@ func PrintHelp(args []string) {
...
 // Start starts running a command
-func Start(args []string) {
+func Start(args []string) (err error) {
 	commandMasterFlags.Reset()
 	rootCmd = NewRootCmd()
 	rootCmd.SetArgs(args)
-	if err := rootCmd.Execute(); err != nil {
-		fmt.Println(rootCmd.UsageString())
+	if err = rootCmd.Execute(); err != nil {
+		//fmt.Println(rootCmd.UsageString())
```
Comment (GMHDBJD) [inline dm/ctl/ctl.go:140]: remove it?

Comment (lance6716) [reply dm/ctl/ctl.go]: https://github.com/pingcap/dm/pull/845/commits/45742af81173d51efc763299ddb08d56a3a046aa

After changing to `RunE`, `rootCmd.Execute()` will return error now. For some error we manually print usage inside command function, so don't print again here (will break some tests)

File: dm/ctl/master/start_task.go [hunk 2]
```diff
@@ -29,34 +30,33 @@ func NewStartTaskCmd() *cobra.Command {
...
 	sources, err := common.GetSourceArgs(cmd)
 	if err != nil {
-		common.PrintLines("%v", err)
 		return
 	}
 
 	removeMeta, err := cmd.Flags().GetBool("remove-meta")
 	if err != nil {
-		common.PrintLines("%v", err)
+		common.PrintLines("error in parse remove-meta")
```
Comment (csuzhangxc) [inline dm/ctl/master/start_task.go:59]: a little suggestion, for `error in parse` error, how about showing the flag as `--remove-meta` or ``` `remove-meta` ```?

Comment (lance6716) [reply dm/ctl/master/start_task.go]: https://github.com/pingcap/dm/pull/845/commits/425157f3d92faa1a42bda39a9e62edc52e8b694b

File: dm/ctl/ctl.go (hunk 1 above)
Comment (lance6716) [reply dm/ctl/ctl.go]: forget to set returned err, fix here https://github.com/pingcap/dm/pull/845/commits/61456ff3b52c870dcf2a80ab365e453dc2c2e14f

**Comment by lance6716** [comment]
/run-all-tests

🤔

**Pull request merged**